tests/cpp_unit_tests/test_main_model.cpp

```diff
@@ -941,6 +941,129 @@ TEST_CASE_TEMPLATE("Test main model", settings, regular_update, cached_update) {
             CHECK(sym_node_2[8].u_pu == doctest::Approx(0.67).epsilon(0.005));
         }
     }
+
+    SUBCASE("Test incomplete input but complete update dataset") {
```

Comment:
@TonyXiang8787 what's the best way to test this for state estimation?

Reply:
made an attempt below. and yes, i know it doesn't make sense to use both a symmetric voltage sensor and an asymmetric one in the same place, but it's just to check that the code in principle works.

EDIT: actually, it does make sense cause they represent measurements, not exact values